components/script/dom/readablestreamdefaultcontroller.rs

```diff
@@ -135,6 +135,7 @@ pub struct ValueWithSize {
 #[allow(crown::unrooted_must_root)]
 pub enum EnqueuedValue {
     /// A value enqueued from Rust.
+    #[allow(dead_code)]
```

Comment:
Yes this is something interesting because of https://github.com/servo/servo/blob/36f6d08e45a54367f2297c740741332705a39778/components/script/dom/readablestreamdefaultcontroller.rs#L560

so we later can either not use jsvals for native values(as a kind of optimization), or remove this.